lib/yeti_logger/custom_formatter.rb

```diff
@@ -0,0 +1,33 @@
+class YetiLogger::CustomFormatter
+  # @param tags [Hash] - maps names of tags to procs that return their value
+  def initialize(tags = {})
+    super()
+    @tags = tags
+  end
+
+  # @param tags [Hash] - maps names of tags to procs that return their value
+  def add_tags(tags)
+    @tags.merge!(tags)
+  end
+
+  # @param severity [String]
+  # @param time [Time] unused
+  # @param progname [String] unused
+  # @param msg [String]
+  def call(severity, time, progname, msg)
+    timestamp = "#{Time.now.iso8601(3)}"
+    pid = Process.pid
+    msg = msg.inspect unless msg.is_a?(String)
+    msg = "#{msg}\n" unless msg[-1] == ?\n
```

Comment:
```suggestion
    msg = "#{msg.inspect}\n" unless msg.is_a?(String) && msg.end_with?("\n")
```

Reply:
Turns out your suggestion is not equivalent with the current code.

Below are logs produced by the current code. The contents of `msg` are to the right of the dash (`-`) character.
![Screenshot 2024-03-01 at 4 17 42 PM](https://github.com/vendasta/yeti_logger/assets/83599192/4b43d078-ddf7-4822-836f-edce4947b261)

And below are logs produced by your suggestion.
![Screenshot 2024-03-01 at 4 18 46 PM](https://github.com/vendasta/yeti_logger/assets/83599192/c5663936-3f15-4b95-bbc6-b71c8c4f6227)

Note the additional quotation marks around the body of the log using your suggested approach.